### DISCUSSION TITLE
More blocks in rich editor

### DISCUSSION THREAD

**Discussion by chahal26** [filamentphp/filament#3088] [Help]
Hi, do we have limited blocks available in rich editor? I want a few more blocks there. How can we achieve that?

**Comment by ryangjchandler** [comment]
@dev-chahal The rich editor field is powered by Trix, so we're limited to what Trix provides.

**Reply by chahal26** [reply]
@ryangjchandler Thanks for the reply. I found a plugin for the filament that helped me there.
